src/Domain/Entity/Group.php

```diff
@@ -4,5 +4,13 @@
 
 abstract class Group extends CoreEntity
 {
+    public function isTlec(): bool
+    {
+        return true;
+    }
 
+    public function isTleo(): bool
+    {
+        return true;
```

Comment:
Do we ever consider groups TLEOs? It's not really a PIPs concept so we can define it as we'd like, but I'd say groups can't be TLEOs or TLECs the way we normally define those